documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx

```diff
@@ -44,33 +49,46 @@ To make it easier for the reader, we use a highlighting line on the left side, w
 </Callout>
 */}
 
+
 <Callout type="info" emoji="ℹ️">
 Nodes bonded with vesting tokens are [not allowed to join rewarded set](https://github.com/nymtech/nym/pull/5129) - read more on [Nym operators forum](https://forum.nymtech.net/t/vesting-accounts-are-no-longer-supported/827).
 </Callout>
 
 
-## Overview
+## Rewards Logic & Calculation
```

Comment:
There is a 'rewards calculation' section toward the bottom of the page already as well as this

Reply:
Since this section seems like an overview, perhaps we revert the section title change? Or modify the 'rewards calculation' subsection header?